Eduplatform_site/account/models.py

```diff
@@ -0,0 +1,33 @@
+
+
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+from .mixins import DateTimeMixinModel
+from .managers import CustomUserManager
+
+
+class User(DateTimeMixinModel,AbstractUser):
```

Comment:
Why you choose an AbstractUser instead of AbstractBaseUser?

Reply:
there are more fields to choose from